tests/sentry/backup/README.md

```diff
@@ -0,0 +1,37 @@
+# Backup tests
+
+This directory tests a number of scenarios where we take an empty database, import a given backup `.json` file, optionally perform some transform, re-export it and validate the diff. The expected result is that only the changes we have made are reflected in the final diff between the original input `.json` that we imported, and the final output `.json` that we exported.
```

Comment:
nit: Line-wrap consistently with other paragraphs in the file?

Reply:
Done.